style.css

```diff
@@ -0,0 +1,4 @@
+h1 {
+  color: red;
```

Comment:
- [ ] Please use a class to style elements, not a general class

Reply:
@MeqdamAlqudah Done!